js/honkalculate_controls.js

```diff
@@ -295,7 +295,11 @@ $(".set-selector").change(function (e) {
 var mode, dtHeight, dtWidth;
 $(document).ready(function () {
 	var url = window.location.href;
-	mode = url.substring(url.indexOf('=') + 1, url.length);
+	if (url.indexOf('=') < 0) {
```

Comment:
Cache search functions if you're reading from them more than once.

Reply:
Is it worth doing that for two function calls? I just took this as the easy way out to avoid having to think of a variable name ("indexOfEqualsSign"? idk)

Reply:
I'd call it `equalsPos`.

But then again, I've written this code millions of times. Good to get into the right habit.